pandas/core/indexes/datetimelike.py

```diff
@@ -157,6 +157,7 @@ def __contains__(self, key: Any) -> bool:
             res = self.get_loc(key)
         except (KeyError, TypeError, ValueError):
             return False
+        return True
```

Comment:
what is this?

Reply:
i think this and the FIXME weren't meant to be committed; will fix

Reply:
updated + all-but-arm-green